src/components/Card/Card.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { ReactNode } from "react";
+import styled, { css } from "styled-components";
+
+export interface CardProps {
+  children: ReactNode;
+  withHover?: boolean;
+}
+
+export const Card = ({ children, ...props }: CardProps) => (
+    <StyledCard {...props}>
+      {children}
+    </StyledCard>
+  );
+
+const StyledCard = styled.section`
```

Comment:
nit: We can add the generic after the element and then we don't need to give the prop type on line 23, makes it a bit cleaner IMO

```suggestion
const StyledCard = styled.section<CardProps>`
```

Reply:
Fixed with [b2d44a](https://github.com/Major-Eth/yearn-web-lib/pull/20/commits/b2d44a7bbff0606b6cda65ba4a8b28e2263cb7f9) with linting